CODEOWNERS

```diff
@@ -1,3 +1,14 @@
+# require @mozilla/data-platform-infra-wg review for everything else
+* @mozilla/data-platform-infra-wg
```

Comment:
Not an issue with this PR per se, but it seems like the @mozilla/data-platform-infra-wg team membership might need to be updated (e.g. @kwindau isn't currently a member).